# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../zaptel/hotfix_noredef_bool.diff
# Copyright (C) 2006 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./xpp/xdefs.h.orig	2006-10-09 09:33:11.000000000 -0400
+++ ./xpp/xdefs.h	2006-10-09 09:52:31.000000000 -0400
@@ -90,7 +90,6 @@
 
 typedef char			*charp;
 typedef unsigned char		byte;
-typedef int			bool;
 typedef struct xbus		xbus_t;
 typedef	struct xpd		xpd_t;
 typedef	struct xpacket_raw	xpacket_raw_t;
